packages/webpack/src/buildtime/emitSes.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     () => {
       // Read the file you want to add to the output
       const sesFile = readFileSync(require.resolve('ses'), 'utf-8')
+      // TODO: instead manually copy to compiler.options.output.path
```

Comment:
why

Reply:
because I didn't find a way to prevent it from being minified :(

Reply:
todo: try telling webpack it's not a source file

Reply:
it seems webpack is choosing to minify it by extension :disappointed:

Reply:
current attempt: fix it at the cost of user astonishment